scripts/alert_emails_report/generate-report.ts

```diff
@@ -0,0 +1,96 @@
+import path from 'path';
+import _ from 'lodash';
+import moment from 'moment';
+import CampaignMonitor from '../alert_emails/campaign-monitor';
+import { getFirestore } from '../alert_emails/firestore';
+import { ALERT_EMAIL_GROUP, toISO8601 } from '../alert_emails/utils';
+import fipsByStateCode from '../what-the-fips/2018-census-fips-codes.json';
```

Comment:
heh...  what-the-fips was a one-off hack I did a while ago.

Can we get this data from the https://github.com/covid-projections/covid-projections/blob/a35fd31c1d48e063b05af8a362c08ee637888054/src/components/MapSelectors/datasets/us_states_dataset_01_02_2020.json (perhaps adding a helper to https://github.com/covid-projections/covid-projections/blob/a35fd31c1d48e063b05af8a362c08ee637888054/src/common/locations.ts)?

Reply:
👌